site/assets/scss/_component-examples.scss

```diff
@@ -320,3 +324,26 @@
     margin-left: 0;
   }
 }
+
+.bd-example-multiple-langs + .bd-clipboard + .highlight {
+  margin-bottom: 0;
+  border-bottom: 0;
+  @include border-bottom-radius(0);
+}
+.highlight + .bd-clipboard + .highlight {
```

Comment:
Can't recall nor check the DOM right now but wouldn't `~` operator help on this, e.g. `.highlight ~ .highlight`?

As long as highlights are contained and scoped to a single example, it should work.

Reply:
They're not bound to a unifying parent, so this would apply to every single highlight afterwards.